src/styles/main.scss

```diff
@@ -1,3 +1,260 @@
-body {
+* {
+  box-sizing: border-box;
+  line-height: 1;
   margin: 0;
+  padding: 0;
```

Comment:
you can put only 'box-sizing: border-box' to the *{}, other styles you can put to the tag body